src/ui/utils/tree.js

```diff
@@ -0,0 +1,31 @@
+export const getObject = async (openmct, domainObject, identifier) => {
+    if (domainObject.identifier.key === identifier.key) {
+        return true;
+    }
+
+    if (!domainObject.composition) {
+        return false;
+    }
+
+    let hasObject = false;
+    for (const composition of domainObject.composition) {
+        hasObject = hasObject
+            || await openmct.objects.get(composition)
+                .then(object => getObject(openmct, object, identifier));
+    }
+
+    return hasObject;
+}
+
+export const searchObjectInTree = async (openmct, identifier) => {
```

Comment:
Is this still needed? I don't think we need to do an exhaustive tree search any more?

Reply:
removed.